index.d.ts

```diff
@@ -254,9 +254,18 @@ declare namespace execa {
 		killed: boolean;
 
 		/**
-		The signal that was used to terminate the process.
+		The name of the signal that was used to terminate the process. For example `"SIGFPE"`.
+
+		If a signal terminated the process, this property is defined and included in the error message. Otherwise it is `undefined`.
 		*/
 		signal?: string;
+
+		/**
+		A human-friendly description of the signal that was used to terminate the process. For example `"Floating point arithmetic error"`.
+
+		If a signal terminated the process, this property is defined and included in the error message. Otherwise it is `undefined`.
```

Comment:
This is not entirely correct as there might be a signal that we don't have a description for.

Reply:
Fixed in cf3161a09dcff8a38fb9f31681c3af077090988a